modules/siddhi-query-api/src/main/java/org/wso2/siddhi/query/api/exception/NoSuchDurationException.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.siddhi.query.api.exception;
+
+/**
+ * Exception class to be used when user provided duration is invalid
+ **/
+public class NoSuchDurationException extends RuntimeException {
```

Comment:
Why is this class defined twice, in two different packages?

Reply:
Removed the duplicated class.

Reply:
Use siddhi app creation exceptions, no need a new exception here